Automattic/woocommerce-payments
# Remove duplication of deposit schedule

Fixes #4453 

#### Changes proposed in this Pull Request

Remove duplication of deposit schedule.

The deposit schedule is duplicated on the current **Payments > Overview** page. The schedule is shown underneath the heading with a detailed description of the schedule e.g. _"Automatic, Automatic, every week on Monday"_, and again in the account details panel showing the deposit status along side a simplified schedule e.g. _"Weekly"_.  

<img width="700" src="https://user-images.githubusercontent.com/1527164/179666563-2f3f39dd-61aa-4dd5-ab7b-19ac83b0cb45.png" />

This PR removes the simplified schedule, replacing the description with `Automatic` if the account currently has an automatic schedule in place for deposits.



### Changes

| Before | After |
| - | - |
| <img width="400" alt="screen-shot-2022-06-16-at-9 34 37-am" src="https://user-images.githubusercontent.com/1527164/179665799-33606eaa-a6f4-4321-a45d-e7b1c4596c41.png"><br><img width="400" alt="screen-shot-2022-06-16-at-9 35 50-am" src="https://user-images.githubusercontent.com/1527164/179665808-ea673f45-8b43-434d-8fa3-1b473a1bcc14.png"><br><img width="400" alt="screen-shot-2022-06-16-at-9 37 08-am" src="https://user-images.githubusercontent.com/1527164/179665822-8d20219a-7914-4073-b9c3-7d54fe7e6da9.png"> | <img width="400" src="https://user-images.githubusercontent.com/1527164/179665282-086d918e-7daf-4734-b6ab-2f7e117d91c2.png" /> |


#### Testing instructions

- Set deposit schedule to `Daily`
- Clear account cache in WCPay ( WooCommerce > Status > Tools > Clear WooCommerce Payments account cache )
- Visit Payments > Overview page. 
- Confirm Deposit schedule detail is shown correctly in "Balance" panel and "✅ Automatic" is show in Account Details Panel
- Repeat for `Weekly` and `Monthly` 

-------------------


**Post merge**
